src/Components/Shifting/ShiftingBoard.tsx

```diff
@@ -168,7 +168,7 @@ export default function ListView({ board, filterProp, formatFilter }: boardProps
                   <i className="fas fa-eye mr-2" /> All Details
                 </button>
               </div>
-              {filter === "TRANSFER IN PROGRESS" &&
+              {filter === "TRANSFER IN PROGRESS" && shift.assigned_facility &&
```

Comment:
show "TRANSFER TO RECEIVING FACILITY" button if assigned_facility is present condition